stix_shifter_modules/symantec_endpoint_security/README.md

```diff
@@ -0,0 +1,700 @@
+                },
+                "3": {
+                    "type": "x-oca-geo",
+                    "name": "Default",
+                    "x_is_on_premises": false
+                },
+                "4": {
+                    "type": "mac-addr",
+                    "value": "11:1X:XX:X1:11:X1"
+                },
+                "5": {
+                    "type": "x-symantec-policy",
+                    "name": "Default Antimalware Policy",
+                    "version": "1"
+                },
+                "7": {
+                    "type": "ipv6-addr",
+                    "value": "xx11::1x1x:11xx:11xx:x1x1",
+                    "resolves_to_refs": [
+                        "4"
+                    ]
+                },
+                "8": {
+                    "type": "ipv4-addr",
+                    "value": "2.2.2.2",
+                    "resolves_to_refs": [
+                        "10"
+                    ]
+                },
+                "9": {
+                    "type": "ipv6-addr",
+                    "value": "xx22::22x2:x0xx:22x2:22xx",
+                    "resolves_to_refs": [
+                        "10"
+                    ]
+                },
+                "10": {
+                    "type": "mac-addr",
+                    "value": "22:X2:22:X2:X2:22"
+                },
+                "11": {
+                    "type": "software",
+                    "name": "Windows Server 2019 Datacenter Edition",
+                    "x_os_type": "Windows"
+                },
+                "12": {
+                    "type": "ipv4-addr",
+                    "value": "3.3.3.3"
+                },
+                "13": {
+                    "type": "user-account",
+                    "user_id": "SYSTEM"
+                }
+            },
+            "number_observed": 1,
+            "last_observed": "2024-05-06T08:33:25Z",
+            "first_observed": "2024-05-06T08:33:25Z"
+        }
+    ],
+    "spec_version": "2.0"
+}
+```
+### Types of Attributes
+
+| Type                      | Description                                | Example                                                                                                            |
+|---------------------------|--------------------------------------------|--------------------------------------------------------------------------------------------------------------------|
+| Dictionary fields         | A key value pair attributes.               | {"user_name": "SYSTEM"}, {"feature_name": "MALWARE_PROTECTION"}                                                              |
+| List of dictionary fields | A list containing one or more dictionaries | "device_networks": [{"ipv4": "1.1.1.1", "ipv6": "xx22::2x2x:22xx:22xx:x2x2", "mac": "11:1X:XX:X1:11:X1" }, { "ipv4": "2.2.2.2", "ipv6": "xx11::1x1x:11xx:11xx:x1x1", "mac": "22:X2:22:X2:X2:22"}] |
+
+### Observations
+- The access token expires 60 minutes after being created.
```

Comment:
Do we have an error message for this.

Reply:
This error won't occur in connector as new token is getting created for every API request. It is just as an observation and it is now removed from ReadMe.